thumbscrew.scpt

```diff
@@ -1,35 +1,35 @@
 -- thumbscrew.scpt
--- usage: thumbscript.scpt <path_to_root_dir> <path_to_keynotePresentation>
+-- usage: thumbscript.scpt <keynotePresentation>
 --
 -- Will create scaled thumbnails of length `thumbSize` in the directory
--- <path_to_root_dir>/<thumbnailDir>/<presentation>
+-- <path_to_keynotePresentation>/<thumbnailDir>
 property thumbSize : 480
-property thumbnailDir: "keynotes"
+property thumbnailDir: "thumbnail"
+
 
 on getImages(f)
   tell application "Finder" to return (files of folder f) as alias list
 end getImages
 
 on run argv
-  set gitRoot to item 1 of argv
-  set tmpFile to item 2 of argv
+  set repoPath to item 1 of argv
   set documentName to item 2 of argv
-  if documentName ends with ".key" then set documentName to text 1 thru -5 of documentName
+  set documentFullName to repoPath & "/" & documentName
 
-  set gitRoot to posix file (POSIX path of (gitRoot as text) & "/" & thumbnailDir) as alias
-  set tmpFile to (POSIX file tmpFile) as alias
+  set keynoteFile to (POSIX file documentFullName) as alias
+  set documentPath to posix file (POSIX path of (do shell script "dirname " & quoted form of documentFullName)) as alias
```

Comment:
I did look at other ways to do this, but everything I found sounded overly complicated. I'll look more.